Describe/DescribeServiceRequestAPI.cs

```diff
@@ -45,6 +45,31 @@ public string uri
             set;
         }
 
+        public string UriTrimmed 
```

Comment:
Can you add a `[JsonIgnore]` attribute here, just so it's clear to people reading the code that we don't want this returned.